src/core/Cline.ts

```diff
@@ -1440,6 +1437,8 @@ export class Cline extends EventEmitter<ClineEvents> {
 							const modeName = getModeBySlug(mode, customModes)?.name ?? mode
 							return `[${block.name} in ${modeName} mode: '${message}']`
 						}
+						default:
+							return `[${block.name}]`
```

Comment:
got this error when pushing 

src/core/Cline.ts:1398:33 - error TS2366: Function lacks ending return statement and return type does not include 'undefined'.

1398     const toolDescription = (): string => {
                                     ~~~~~~


Found 1 error in src/core/Cline.ts:1398